### ISSUE TITLE
Pilgrim locations for King's Men

### ISSUE THREAD

**Issue by Shammien** [MattTheLegoman/RealmsInExile#478]
Pelargir is already done. So here are the rest:

**Haven of Umbar**

I furl my ship’s sails as I approach the docks of the grand Haven of Umbar. Even my mighty ship seems but a tiny toy boat amongst the venerable towers and structures of Adûnaic stonework.

As I disembark upon the pier at the southern side of the harbor, I gaze upon the magnificent Palaces of the Sea-Lords of old Anadûnê. Further inside the city, I see the remnants of Tarma Tar Calion, the memory of the last great king of Anadûnê, Ar-Pharazôn. Will I be able to bring our people to the same heights I wonder?

I continue on through the busy streets of the Market District and observe the faded statues of the great Umbarean kings of the past - amongst them I see Herumor, Fuinur and even the righteous King Castamir himself.

As the sun sets, I return to my ship filled with hope and ambition for my future. I am certain, by the grace of the Adûnaic kings of old, I shall bring our people to new heights. One day, my heirs will walk the streets of Umbar gazing upon statues of my likeness.

(986 characters)

**Târikêth an-Kharat**

My retinue and I arrive exhausted in Târikêth an-Kharat by horse. As I pass over the hilly farmlands, my exhaustion comes to an abrupt end as I lay my eyes upon the imposing walls of Zadan an-Adûn - the jewel of Târikêth an-Kharat.

Towering over the walls, I see a magnificent work of Adûnaic stonework, a remarkable fortress with a red hue. I enter the city through the main gates. As I wander the cobbled streets, I inspect the impeccable stonework of the city. Surely this is proof that the Anbalûkkhòri are true descendants of Anadûnê.

Finally I enter the walls of the Red Citadel itself. Within the imposing citadel lies the seat of the Anbalûkkhòri throne. The halls are draped with pompous artwork, as well as luxurious goods and relics of old Anadûnê displayed proudly in the many halls of the keep.

As I leave the city and trot my way home, I ponder over the many great Adûnai, loyal to the King’s Men of old, that ruled from this seat of power in ancient times.

(969(nice) characters) 

**Hazaj Tollin**

Ah, Hazaj Tollin… The first Adûnaic settlement in Bellakar, established by the Adûnai in the early years of the Second Age during the reign of the ninth king of Anadûnê, the glorious Tar-Súrion.

I dismount my horse and walk along the city premises for hours, as I marvel at the Arch of Ciryandur beyond the city gates. I recall the namesake of this Arch; the honorable Adûnaic captain of the that discovered coast Bellakar and established Hazaj Tollin. I inspect the intricate details along the surface of the arch, which unlike the rest of the city remained intact after the great waves of the Akallabêth washed over this place.

The magnificent stoneworks of my ancestors are truly a sight to behold. I close my eyes and ponder over the noble deeds of my forefathers - the establishment of this settlement and the subjugation and subsequent ennoblement of the Haradrim. I am filled with a sense of pride and awe as I mount my horse and leave the city's boundaries.

(959 characters)

**Korlea**

I arrive at the Korelan harbor by boat. The inlet leading to the harbor is teeming with all kinds of ships, both great and small - magisterial mercantile ships, fishing boats of all sizes, military vessels and lastly; the pretentious leisure ships docked at the noble’s district pier.

Korlea is a melting pot of the indigenous cultures, but beneath all the rabble, I can see the architecture and chiseled stucco of Adûnaic craft. I can sense the proud heritage of the Adûnaim as I explore the nooks and crannies of the city. Korlea was once a proud and fair city of the most loyal King’s Men of old.

I stroll through the various districts until sundown, before I decide to return to my ship for a good night's sleep. My dreams take me to the near future of Korlea under firm King’s Men control. I see the silhouette of a kingly Adûnaic figure, sitting upon a Korelan throne while the masses cheer. The silhouette reminds me of… Me. The dream is intense, but I feel at ease.

(971)

**Comment by jj248** [comment]
Added - these should now be available to non-Good faiths with holy sites in these places.  Note that you shouldn't be able to go to a pilgrimage to these places if you have an evil faith and the holder has a good faith.  e.g. Hazaj Tollin shouldn't be available at game start unless taken over by non-Good characters.

Ready to test.

**Comment by savagesach** [comment]
I tested the issue as umbar and I was only able to go to places controlled by evil faiths. Taking over felaya with console commmands did allow me to go on a pilgrimage to hazaj tollin, but I can't understand why taking over bellakar and gondor didn't allow me to go on a pilgrimage to pelargir or korlea.
![image](https://user-images.githubusercontent.com/86676697/221401304-db3f15ba-7aed-418f-a141-911c0fe13a82.png)
![image](https://user-images.githubusercontent.com/86676697/221401362-d3e33f89-f111-4dce-b9d9-b72d2299232c.png)
![image](https://user-images.githubusercontent.com/86676697/221401378-372d0bf9-b7d5-4efd-82fe-a5ed656126ac.png)

**Comment by savagesach** [comment]
okay I figured out the problem. The way the game calculates who owns the holy sites, it doesn't look at the top liege. it instead looks at who owns the actual counties. I was therefore only able to pilgrimage to pelargir or korlea after I revoked the titles, which is kinda stupid when it's literally in my nation.
![image](https://user-images.githubusercontent.com/86676697/221401553-b6db5461-b62e-4c3e-8294-76051abc1a48.png)
